ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml

```diff
@@ -319,6 +319,40 @@ tests:
         requests:
           cpu: 100m
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-periodic-pre
+  optional: true
+  run_if_changed: .*periodic.*
```

Comment:
You don't want the "or openstack" clause like the others have?

Reply:
Hah. I was cribbing from the only example which does not have that (AWS).